static/app/views/performance/transactionSummary/transactionSpans/suspectSpansTable.tsx

```diff
@@ -47,8 +47,10 @@ export default function SuspectSpansTable(props: Props) {
     description: suspectSpan.description,
     totalCount: suspectSpan.count,
     frequency:
+      // Due to how Clickhouse retrieves data via approximations, sometimes the frequency value can exceed the total number of spans.
+      // This can result in a ratio greater than 1, so use Math.min here to avoid those cases
```

Comment:
The explanation here isn't quite right.

```suggestion
      // Frequency is computed using the `uniq` function in ClickHouse.
      // Because it is an approximation, it can occasionally exceed the number of events.
```